### PR TITLE
Register & Login page styles aligned

### PR DESCRIPTION
**Pull request by SteffenDCI**
Tried to make it look better with my modest means.

### CODE REVIEW

File: src/pages/login/Login.jsx [hunk 1]
```diff
@@ -41,9 +41,10 @@ export function Register() {
         <header className="App-header">
           <div className="App-logo-container"></div>
           <div className="App-title-container">
-            <h2 className="App-title">Register your account here</h2>
+          <h1 className="App-title" style={{ fontFamily: 'IBM Plex mono', letterSpacing: '2px', fontSize: '2vw'}}>Become our newest member | Same Time Same Place</h1>
```
Comment (Termuellinator) [inline src/pages/login/Login.jsx:44]: I think we should use the same font for login and registration

File: src/App.css [hunk 2]
```diff
@@ -1,3 +1,5 @@
+@import url('https://fonts.googleapis.com/css2?family=Poppins:wght@300;400;500;600&display=swap');
```
Comment (Termuellinator) [inline src/App.css:1]: I'm not the biggest fan of using google fonts to be honest, as this is know to be used by google to track users even if no other google services are used.

**Review by Termuellinator — CHANGES_REQUESTED** [review]
Thanks for your work!
I think it looks quite nice already, though the font should be consistent between login and register (and not a dynamically imported google font).
Also, the Heading is not consistently aligned between login and register.

File: src/App.css (hunk 2 above)
Comment (SteffenDCI) [reply src/App.css]: Thanks, I learned something new today,

**Pull request merged**